nexus/posts/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from . import models
+
+
+class PostSerializer(serializers.ModelSerializer):
+    posted_by = serializers.ReadOnlyField(source='posted_by.email')
+    created_at = serializers.SerializerMethodField()
+    modified_at = serializers.SerializerMethodField()
+    posted_time = serializers.SerializerMethodField()
+
+    class Meta:
+        model = models.Post
+        fields = ['id', 'created_at', 'modified_at', 'posted_by', 'posted_at',
+                  'posted_time', 'content_type', 'image',
+                  'text', 'is_approved', 'is_posted']
+        ReadOnlyField = ['id', 'created_at', 'modified_at']
```

Comment:
This is not a valid property. Where did you pick this from?

Reply:
This happened by mistake of hinting. I've fixed it to `read_only_fields`.